src/components/MakeSelection/MakeSelection.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "../SelectItem/SelectItem.css";
+
+const MakeSelection = (props)=>{
+    return (
+      <div>
+        <select className="border-control" onChange={props.onChangeHandler}>
```

Comment:
Please provide the custom css class too.

For example, I want add my own class, `<MakeSelection className="custom" />`